packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -344,6 +346,13 @@ export function printBindings(
 		});
 	}
 
+	if (images !== undefined) {
+		output.push({
+			name: friendlyBindingNames.images,
+			entries: [{ key: "Name", value: images.binding }],
+		});
+	}
```

Comment:
This should have some explanation of whether or not it's local, using e.g. `addLocalSuffix(...)`. Although cc @andyjessop who's changing this around a bit